apps/dot_ext/loggers.py

```diff
@@ -20,7 +20,8 @@
 SESSION_AUTH_FLOW_TRACE_KEYS = ['auth_uuid', 'auth_client_id', 'auth_grant_type', 'auth_app_id',
                                 'auth_app_name', 'auth_app_data_access_type',
                                 'auth_pkce_method', 'auth_crosswalk_action',
-                                'auth_share_demographic_scopes', 'auth_require_demographic_scopes']
+                                'auth_share_demographic_scopes', 'auth_require_demographic_scopes',
+                                'language']
```

Comment:
@sharonfruit Should change the naming of this to match the grouping with the other `auth_*` items being traced as part of the flow.

```suggestion
                                'auth_language']
```

Reply:
Will do for all, thanks Dave!